src/compliment/sources/class_members.clj

```diff
@@ -114,21 +129,27 @@
 (defn members-candidates
   "Returns a list of Java non-static fields and methods candidates."
   [prefix ns context]
-  (when (class-member-symbol? prefix)
-    (let [prefix (subs prefix 1)
-          inparts? (re-find #"[A-Z]" prefix)
-          klass ^{:lite nil} (try-get-object-class ns context)]
-      (for [[member-name members] (get-all-members ns klass)
-            :when (and (if inparts?
-                         (camel-case-matches? prefix member-name)
-                         (.startsWith ^String member-name prefix))
-                       ^{:lite true}
-                       (or (not klass)
-                           (some (fn [m] (.isAssignableFrom (.getDeclaringClass ^Member m) klass))
-                                 members)))]
-        {:candidate (str "." member-name)
-         :type (if (instance? Method (first members))
-                 :method :field)}))))
+  (when-let [[klass-name prefix] (class-member-symbol? prefix)]
+    (let [prefix      (str/replace prefix #"^\." "")
+          qualified?  (not-empty klass-name)
```

Comment:
Was there any reason you changed this from returning `nil` for klass-name and testing with `some?`? I liked that variant better.